tools/fp20compiler/ts1.0_inst_list.cpp

```diff
@@ -38,6 +38,14 @@ InstList& InstList::operator+=(InstPtr t)
     return *this;
 }
 
+static unsigned int float_to_uint(float value)

```

Comment:
nit:  I'm not 100% sure about `ToUint` because it could imply something like `(float)1.0f` being turned into `(unsigned int)1`. Hence my initial proposal to something like "Raw".